conda_smithy/feedstock_content/build-locally.py

```diff
@@ -16,13 +17,22 @@ def setup_environment(ns):
         os.environ["BUILD_WITH_CONDA_DEBUG"] = "1"
         if ns.output_id:
             os.environ["BUILD_OUTPUT_ID"] = ns.output_id
+    if "MINIFORGE_HOME" not in os.environ:
+        os.environ["MINIFORGE_HOME"] = os.path.join(
+            os.path.dirname(__file__), "miniforge"
```

Comment:
This would produce `~/miniforge/miniforge3` as the install location for miniforge if not present. Shouldn't the line here rather omit `miniforge`?

Reply:
It is correct. `bash Miniforge.sh -b -p $MINIFORGE_HOME` will install miniforge into `$MINIFORGE_HOME` without an additional diretory inside.